packages/playwright-test/src/plugins/vitePlugin.ts

```diff
@@ -150,7 +153,8 @@ export function createPlugin(
       const isAddressInfo = (x: any): x is AddressInfo => x?.address;
       const address = previewServer.httpServer.address();
       if (isAddressInfo(address))
-        process.env.PLAYWRIGHT_VITE_COMPONENTS_BASE_URL = `http://localhost:${address.port}/${relativeTemplateDir}/index.html`;
+        process.env.PLAYWRIGHT_VITE_COMPONENTS_BASE_URL = `http://localhost:${address.port}`;
```

Comment:
Where is this used?

Reply:
For baseURL to work, you can either assign it in mount.ts or say PLAYWRIGHT_TEST_BASE_URL = ... here and the packages/playwright-test/src/index.ts will pick it up. Latter is a bit of a hack, but given that it is covered with the test, we can do that.

Reply:
Oh, that's a lot easier than i thought it would be. I replaced `PLAYWRIGHT_VITE_COMPONENTS_BASE_URL` with `PLAYWRIGHT_TEST_BASE_URL` and used it in `mount()`. If I understand correctly, there is already a test for this, right?